package.json

```diff
@@ -91,6 +91,7 @@
     "redux-thunk": "2.2.0",
     "rnnoise-wasm": "github:jitsi/rnnoise-wasm.git#566a16885897704d6e6d67a1d5ac5d39781db2af",
     "rtcstats": "github:jitsi/rtcstats#v6.2.0",
+    "stackblur-canvas": "^2.3.0",
```

Comment:
Please drop the caret (^)

Reply:
https://github.com/jitsi/jitsi-meet/pull/7473/commits/cc3231813be0920417f28aef1895d53177ed4616